test/extensions/filters/network/common/fuzz/network_readfilter_fuzz_test.cc

```diff
@@ -9,41 +9,86 @@
 #include "test/fuzz/fuzz_runner.h"
 #include "test/test_common/test_runtime.h"
 
+#include "libprotobuf_mutator/src/libfuzzer/libfuzzer_macro.h"
+#include "src/text_format.h" // from libprotobuf_mutator/
+
 namespace Envoy {
 namespace Extensions {
 namespace NetworkFilters {
 
-DEFINE_PROTO_FUZZER(const test::extensions::filters::network::FilterFuzzTestCase& input) {
+void ensuredValidFilter(unsigned int seed, envoy::config::listener::v3::Filter* config) {
```

Comment:
Should the seed be passed here, or some random number?

Reply:
The seed changes for every invocation of the surrounding function. So it is at least not constant. But I can also use the allocated random number generator. What do you prefer?

Reply:
Seed is typically used to initiate a series of random numbers.
Using the same seed in multiple places slightly limits the ability to test different scenarios.